src/raw_decompiled/HEROES2W.h

```diff
@@ -2958,8 +2958,7 @@ struct heroMapExtra
   int experience;
   char couldBeHasSecondarySkills;
   char secondarySkills[8];
-  char firstSecondarySkillLevel;
-  char secondSecondarySkillLevel;
+  char secondarySkillLevel;
```

Comment:
Uhhh.....why did you delete that other field?

You can't change the size of this structure; doing so can prevent us from loading maps.

Reply:
This...is out of date.